index.html

```diff
@@ -4,7 +4,7 @@
 <div class="jumbotron" style="background-image: url('img/bg.jpg');">
     <div class="container">
         <h1>Welcome to techcorridor.io</h1>
-        <p>TechCorridor.io is a developer group in Iowa City and Cedar Rapids, IA. If you’re interested in programming, the intertubes, and whatnot, this is the group for you.</p>
+        <p>TechCorridor.io is a group of tech professionals and hobbyists based in the Iowa City and Cedar Rapids, IA area. Our hope is that the Tech Corridor will allow members to easily host and attend meetups, spread the word about awesome local events, and ensure anyone new to the scene can easily get in touch with the right people. If you’re interested in computers, programming, or technology of any sort, we'd love to meet you.</p>
```

Comment:
Per our Slack conversation, let's move the middle two sentences to another place (like `about.html`), since this is meant as a quick introduction.

We should also consider putting a link at the end of this paragraph...

> TechCorridor.io is a group of tech professionals and hobbyists based in the Iowa City and Cedar Rapids, IA area. If you’re interested in computers, programming, or technology of any sort, we'd love to meet you. [Learn more...](http://techcorridor.io/about.html)